Tasks/Gradle/sonarqube-common/sonarqube-common.ts

```diff
@@ -205,16 +212,47 @@ function getOrCreateSonarQubeStagingDirectory(): string {
     return sqStagingDir;
 }
 
+// Creates a string containing Markdown of the SonarQube quality gate status for this project.
+function createQualityGateStatusSection(taskReport: TaskReport): Q.Promise<string> {
+    if (!tl.getBoolInput('sqAnalysisWaitForAnalysis')) {
+        // Do not create a quality gate status section if not waiting for the server to analyse the build
+        console.log(tl.loc('sqCommon_NotWaitingForAnalysis'));
+        return Q.when<string>(null);
+    }
+
+    // Wait for analysis to finish so that we can get analysis information from the SonarQube server
+    var timeout:number = 60;
```

Comment:
@bgavrilMS : Do I recall that the default on the MSBuild SonarQube task is 300 seconds / 1 second retry?